src/NuGet.Services.EndToEnd/Support/PushedPackagesFixture.cs

```diff
@@ -46,6 +36,7 @@ public class PushedPackagesFixture : CommonFixture
         private IGalleryClient _galleryClient;
 
         private static readonly string SymbolsProjectTemplateFolder = Path.Combine(Environment.CurrentDirectory, "Support", "TestData", "E2E.TestPortableSymbols");
+        private static readonly string DotnetToolProjectTemplateFolder = Path.Combine(Environment.CurrentDirectory, "Support", "TestData", "E2E.DotnetTool");
```

Comment:
Nit: E2E.TestDotnetTool

Reply:
Good idea! That will help users if they happen to see this package.

Reply:
Actually this the symbols package is called `E2E.SymbolsPackage` so the `E2E.DotnetTool` folder name matches the actual package name.